docs/source/handling_advanced_stypes/handle_heterogeneous_stypes.rst

```diff
@@ -0,0 +1,242 @@
+Handling Heterogeneous Stypes
+=============================
+
+:pyf:`PyTorch Frame` supports many heterogeneous :obj:`stypes<torch_frame.stype>`, including but not limited to :class:`~stype.multicategorical`, :class:`~stype.timestamp` and :class:`~stype.embedding`.
+In this tutorial, we will show you a simple example of handling heterogeneous data types with :pyf:`PyTorch Frame`.
+
+.. contents::
+    :local:
+
+Handling Heterogeneous Columns
+------------------------------
+First, let us create a sample dataset with many different stypes.
+
+.. code-block:: none
+
+    import random
+
+    import numpy as np
+    import pandas as pd
+
+    # Numerical column
+    numerical = np.random.randint(0, 100, size=10)
+
+    # Categorical column
+    simple_categories = ['Type 1', 'Type 2', 'Type 3']
+    categorical = np.random.choice(simple_categories, size=100)
+
+    # Timestamp column
+    time = pd.date_range(start='2023-01-01', periods=100, freq='D')
+
+    # Multicategorical column
+    categories = ['Category A', 'Category B', 'Category C', 'Category D']
+    multicategorical = [
+        random.sample(categories, k=random.randint(0, len(categories)))
+        for _ in range(100)
+    ]
+
+    # Embedding column (assuming an embedding size of 5 for simplicity)
+    embedding_size = 5
+    embedding = np.random.rand(100, embedding_size)
+
+    # Create the DataFrame
+    df = pd.DataFrame({
+        'Numerical': numerical,
+        'Categorical': categorical,
+        'Time': time,
+        'Multicategorical': multicategorical,
+        'Embedding': list(embedding)
+    })
+
+    df.head()
+    >>>
+        Numerical Categorical       Time                                  Multicategorical                                          Embedding
+    0         44      Type 2 2023-01-01              [Category D, Category A, Category B]  [0.2879910043632805, 0.38346222503494787, 0.74...
+    1         47      Type 2 2023-01-02  [Category C, Category A, Category B, Category D]  [0.0923738894608982, 0.3540466620838102, 0.551...
+    2         64      Type 2 2023-01-03                          [Category D, Category C]  [0.3209972413734975, 0.22126268518378278, 0.14...
+    3         67      Type 1 2023-01-04                          [Category C, Category A]  [0.2603409275874047, 0.5370225213757797, 0.447...
+    4         67      Type 2 2023-01-05                                      [Category A]  [0.46924917399024213, 0.8411401297855995, 0.90...
+
+
+Now let's load the :class:`pandas.DataFrame` into :class:`~torch_frame.data.Dataset` class.
```

Comment:
```suggestion
Now let's load the :class:`pandas.DataFrame` into :class:`torch_frame.data.Dataset` class so that we have :class:`~torch_frame.data.tensor_frame.TensorFrame` representation of the :class:`pandas.DataFrame`.
```